sounds/migrations/0037_soundanalysis_extractor_version.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2021-02-04 20:49
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('sounds', '0036_sound_uploaded_with_bulk_upload_progress'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='soundanalysis',
+            name='extractor_version',
+            field=models.CharField(db_index=True, default='1.0.0', max_length=255),
```

Comment:
Any specific reason for setting `db_index=True`?

Reply:
As in the instructions it is specified that extractor_version field should be "of the same type as the extractor field", I just copied it as it is, without digging into it. How do you decide if a field should have a database index? Only if it is a meaningful field that is likely to be searched?

Reply:
Yes, having an index in DB improves performance a lot, but at the cost of the DB taking more space (in disk and in memory). In this case it might be worth adding however because we might de doing lots of queries like "give me all sounds with analyzer X and version < Y". In fact, we'll have to make sure that version fields are comparable, so maybe we want to use integers here instead to make it easier. We'll have to think about that.